README.md

```diff
@@ -4,4 +4,15 @@ A collection of scripts useful for hosting/managing Chicago Python User Group (C
 
 Scripts:
  - challenge.grouper: creates groups of participants based on an "experience level."
- - meetup.meetup: Meetup endpoint to track group members/event attendees.
\ No newline at end of file
+ - meetup.meetup: Meetup endpoint to track group members/event attendees.
+
+## Set up Environment
+
+1. Setup and activate Python version environment
+1. `pip install -r requirements.txt`
+
+## Running Challenge Helper
+
+1. `python challenge_helpers/grouper.py`
+1. Use `add`, `bulk add`, or `google add` commands to add folks to Project Night roster
```

Comment:
Would you mind adding details to the readme about expected columns in the csvs for bulk add and google add?